src/NzbDrone.Core/Download/Clients/qBittorrent/QBittorrentTorrent.cs

```diff
@@ -0,0 +1,24 @@
+﻿using System;
+
+namespace NzbDrone.Core.Download.Clients.QBittorrent
+{
+    public class QBittorrentTorrent
+    {
+        public String Hash { get; set; } // Torrent hash
```

Comment:
I won't comment everywhere since there are a number of cases, but please update those as well.